packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPAVFactory.m

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <AVFoundation/AVFoundation.h>
+
+#import "FVPAVFactory.h"
```

Comment:
This should be before any other import, per the style guide.

Reply:
It should also be relative-path-based like the other imports.

Reply:
I think the main (if not the only) reason for using the path-based approach was [this comment](https://github.com/flutter/packages/pull/6675/#discussion_r1591210702), and it was only related to `messages.g.h` . But I agree - better keep it consistent. I used path-based imports in implementation files, but, if you prefer, I think we might also consider using the short import and then just have a path-based import for `messages.g.h`, and a comment there explaining the reasoning behind it